### ISSUE TITLE
Yet another LUA error - Dragonflight Pre-patch

### ISSUE THREAD

**Issue by fraerie** [Kiatra/ChocolateBar#21]
35x bad argument #3 to '?' (Usage: local texture = self:CreateTexture([name, drawLayer, templateName, subLevel]))
[string "=[C]"]: in function `CreateTexture'
[string "@Interface/AddOns/ChocolateBar/Core.lua"]:586: in function <Interface/AddOns/ChocolateBar/Core.lua:580>
[string "@Interface/AddOns/ChocolateBar/Core.lua"]:593: in function `GetPointer'
[string "@Interface/AddOns/ChocolateBar/Bar.lua"]:390: in function `Drag'
[string "@Interface/AddOns/ChocolateBar/DragAndDrop.lua"]:74: in function `Start'
[string "@Interface/AddOns/ChocolateBar/Chocolate.lua"]:343: in function <Interface/AddOns/ChocolateBar/Chocolate.lua:316>

Locals:
(*temporary) = "bad argument #3 to '?' (Usage: local texture = self:CreateTexture([name, drawLayer, templateName, subLevel]))"

**Comment by Zwixx** [comment]
Comment out the Lines 112-116, 56, 71 in Jostle.lua. This works for me so far.

**Comment by fraerie** [comment]
> I try to correct this and made a Pull Request. Opened #22

I have double checked that I have all the changes that are included in the pull requests and the jostle.lua file linked elsewhere and I am still getting this error:

87x bad argument #3 to '?' (Usage: local texture = self:CreateTexture([name, drawLayer, templateName, subLevel]))
[string "=[C]"]: in function `CreateTexture'
[string "@Interface/AddOns/ChocolateBar/Core.lua"]:586: in function <Interface/AddOns/ChocolateBar/Core.lua:580>
[string "@Interface/AddOns/ChocolateBar/Core.lua"]:593: in function `GetPointer'
[string "@Interface/AddOns/ChocolateBar/Bar.lua"]:390: in function `Drag'
[string "@Interface/AddOns/ChocolateBar/DragAndDrop.lua"]:74: in function `Start'
[string "@Interface/AddOns/ChocolateBar/Chocolate.lua"]:343: in function <Interface/AddOns/ChocolateBar/Chocolate.lua:316>

Locals:
(*temporary) = "bad argument #3 to '?' (Usage: local texture = self:CreateTexture([name, drawLayer, templateName, subLevel]))"

**Comment by Zwixx** [comment]
I don't know what changes are you made on the jostile beside mine, but you don't need them. Only the changes made in my pull request are needed.
Line 386 needs to be: local arrow = pointer:CreateTexture(nil, "BACKGROUND")
Do you changed it from DIALOG to BACKGROUND?

**Comment by fraerie** [comment]
yup - made that change and the change colour parameters in the Chocolate.lua in the pull requests added by yourself

I have even pulled down a fresh copy of the zip and made the changes again in case I messed something up the first time

**Comment by Zwixx** [comment]
I found a error, if the minimap is in the standard position. I get an Error. I look onto this.

**Comment by Zwixx** [comment]
Added a fix fpr the nil-error.

**Comment by Kiatra** [comment]
Fixed with v3.9
